src/reference/asciidoc/si-kafka.adoc

```diff
@@ -681,6 +681,47 @@ public void in(@Header(KafkaHeaders.RECEIVED_MESSAGE_KEY) String key,
 ----
 ====
 
+[[streams-integration]]
+==== Calling a Spring Integration flow from a `KStream`
+
+You can use a `MessagingTransformer` to invoke an integration flow from a `KStream`:
+
+====
+[source, java]
+----
+@Bean
+public KStream<Integer, String> kStream(StreamsBuilder kStreamBuilder,
+        MessagingTransformer<String, String, String> transformer)  transformer) {
+    KStream<Integer, String> stream = kStreamBuilder.stream(STREAMING_TOPIC1);
+    stream.mapValues((ValueMapper<String, String>) String::toUpperCase)
+            ...
+            .transform(() -> transformer)
+            .to(streamingTopic2);
+
+    stream.print(Printed.toSysOut());
+
+    return stream;
+}
+
+@Bean
+@DependsOn("flow")
+public MessagingTransformer<String, String, String> transformer)
+        transformer(MessagingFunction function) {
```

Comment:
Looks like typo with duplicated `transformer`.

An interface in the `IntegrationFlows.from()` becomes as a bean with the name of the flow plus `.gateway` suffix.
so, it can be used in the `@Qualifier` for such an autowired parameter.